packages/xdl/static/loading-page/index.html

```diff
@@ -248,6 +266,24 @@
   </div>
   
   <script>
+  const alertElement = document.getElementById("alert");
+
+  const showAlert = (isExpoGo) => {
+    if (isExpoGo) {
+      alertElement.innerHTML = 'It seems that deep-link into your application failed. Make sure that you have the <b>Expo Go</b> installed.' + 
```

Comment:
```suggestion
      alertElement.innerHTML = 'Unable to open in Expo Go. Please install <b>Expo Go</b> to continue.' + 
```
and link to https://expo.dev/client